services/madoc-ts/src/frontend/shared/atoms/CanvasStatus.tsx

```diff
@@ -30,9 +32,14 @@ export const CanvasStatusItem = styled.div<{ $status: number }>`
 `;
 
 export const CanvasStatus: React.FC<{ status: number }> = ({ status }) => {
+  const { t } = useTranslation();
+  const tooltip = status == 3 ? t('Completed') : (status == 2 || status == 1 ? t('You are currently working on this manifest') : t('Available'));
```

Comment:
> You are currently working on this manifest
This may not always be in the context of a manifest. Could we change it something more generic, maybe "You are working on this"